src/main.rs

```diff
@@ -258,6 +260,13 @@ fn main() {
             }
         }
 
+        "clean" => {
+            match fs::remove_dir_all(&config.dest) {
```

Comment:
@rjgoldsborough 

my comment was misleading, sorry for that
i tested on version 0.3, latest master creates dest dir automatically indeed